### DISCUSSION TITLE
Epic Player Support Alternative

### DISCUSSION THREAD

**Discussion by cosmicchicken34** [gameserverapp/Platform#259] [Suggestions]
So obviously we don't have the API for Epic accounts yet so we can't sign in via epic. I found an alternative solution (which I got from tebex.io): When a player goes to the checkout, they can optionally input their Epic ID, which would then replace their Steam ID. I personally don't want to switch to Tebex just to use this function so it would be nice to be able to give all my cross-play players the same store functions.

**Comment by mvaessen** [comment]
This is now added
